### PR TITLE
[python/en] Range function arguments.

### PR DESCRIPTION
**Pull request by srirams6**
Range function with start and stop.

### CODE REVIEW

**Comment by levibostian** [comment]
Could you add the original Python doc changes to this PR as well? Thanks!

**Comment by levibostian** [comment]
Sorry it is a pain we have 2 Python docs. Thanks for the contributions!n

**Comment by srirams6** [comment]
@levibostian  Done :)

File: python.html.markdown [hunk 1]
```diff
@@ -347,6 +347,18 @@ for i in range(4):
...
+"range(lower, upper)" returns a list of numbers
+from the lower number to the upper number
+prints:
+    4
+    5
+    6
+    7
+"""
+for i in range(4, 8):
+    print(i)
```
Comment (geoffliu) [inline python.html.markdown]: Ah but this is Python 2. `print i` please. :)

Comment (srirams6) [inline python.html.markdown]: The whole doc uses the print_function from future. This isn't a good way to teach/learn python2 in my opinion, but I was just sticking to the format of the existing doc.

Comment (geoffliu) [inline python.html.markdown]: Good call, I hadn't realized. Thanks for the PR, and I see you've opened an issue about print statements.

**Pull request merged**